spec/javascript/utils/form/validations.test.js

```diff
@@ -160,15 +160,20 @@ describe('validate', () => {
         expect(validate.isValidCurrency(VALIDATION_MSG)('$2000.53')).toEqual(undefined)
         expect(validate.isValidCurrency(VALIDATION_MSG)('2,000')).toEqual(undefined)
       })
-      // TODO: Enhance currency validation to exclude these cases.
-      test('when any string with $s and ,s is entered', () => {
-        expect(validate.isValidCurrency(VALIDATION_MSG)('$2000$')).toEqual(undefined)
-        expect(validate.isValidCurrency(VALIDATION_MSG)('$2,000,')).toEqual(undefined)
-        expect(validate.isValidCurrency(VALIDATION_MSG)('$')).toEqual(undefined)
+      // Maybe validation should be enhanced for this case?
```

Comment:
not sure. Users are prevented from entering these values at least on Chrome

Reply:
@akegan yep it's across all browsers, (it happens in the `identity()` function in `Field.js`. I'll just update the comment to explain why we don't care about that case